client/puppieslist.js

```diff
@@ -2,33 +2,8 @@ import React from 'react'
 import {Link} from 'react-router-dom'
 
 const Puppieslist = (props) => {
-  const puppies = [{
-    id: 1 , 
-    name: 'Bob',
-    age: 1,
-    bio: 'I like to dig',
-  }, {
-    id: 2 , 
-    name: 'Kit',
-    age: 2,
-    bio: 'I like to catch',
-  }, {
-    id: 3 , 
-    name: 'Jay',
-    age: 3,
-    bio: 'I like to bark',
-  }, {
-    id: 4 , 
-    name: 'Tom',
-    age: 2,
-    bio: 'I like to sleep',
-  }, {
-    id: 5 , 
-    name: 'Aly',
-    age: 0,
-    bio: 'I like to snuggle',
-  }];
-  
+  const puppies = props.puppieslist
```

Comment:
Your `AllPups` was rendering `PuppiesList` with a `puppieslist` prop. You needed to use this prop to actually see the pups coming from the backend (handed down by `AllPups`).